dev-java/core-specs-alpha/core-specs-alpha-0.1.24.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+JAVA_PKG_IUSE="test"
+
+inherit java-pkg-2 java-ant-2
+
+EGIT_REF="43815fc"
+
+MY_PN=${PN//-/.}
+MY_P=${MY_PN}-${PV}
+
+DESCRIPTION="A Clojure library with specs to describe Clojure core macros and functions."
+HOMEPAGE="https://clojure.org/ https://github.com/clojure/core.specs.alpha"
+SRC_URI="https://github.com/clojure/${MY_PN}/archive/${MY_P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="EPL-1.0"
+SLOT="0.1"
+KEYWORDS="~amd64 ~x86 ~x86-linux"
+IUSE=""
+RESTRICT="test" # patches welcome to fix the test
```

Comment:
This should rather contain a comment why it's restricted.

Reply:
Was a left over from copy/paste, removed instead.